fleetspeak/src/server/proto/fleetspeak_server/admin.proto

```diff
@@ -53,6 +53,25 @@ message DeletePendingMessagesRequest {
   repeated bytes client_ids = 1;
 }
 
+message GetPendingMessagesRequest {
+  repeated bytes client_ids = 1;
+  uint64 offset = 2;
+  uint64 limit = 3;
+  bool want_data = 4;
+}
+
+message GetPendingMessagesResponse {
+  repeated fleetspeak.Message messages = 2;
```

Comment:
Given that it's a new proto, pease use index `1` for the field.

Reply:
Done.